### ISSUE TITLE
`:before` and `:after` pseudo elements do not work.

### ISSUE THREAD

**Issue by eopb** [rebo/seed_style_preview#2]
Example code.
```rust
// `view` describes what to display.
pub fn view(model: &state::Model) -> impl View<updates::Msg> {
    div![
        s().after("content to put after"),
        nav(model),
        button![model, ev(Ev::Click, |_| updates::Msg::Increment),],
        title_card(model),
        footer(model)
    ]
}
```
Debug message from browser console.
```
"error inserting style: " JsValue(SyntaxError: An invalid or illegal string was specified
init/imports.wbg.__wbg_insertRule_c0373a211f90c126<@http://localhost:8080/pkg/package.js:473:35
handleError/<@http://localhost:8080/pkg/package.js:254:22
web_sys::features::gen_CssStyleSheet::CssStyleSheet::insert_rule_with_index::hbf13fef597e4c9b3@http://localhost:8080/pkg/package_bg.wasm:wasm-function[913]:0xf1822
seed_style_preview::style::add_css_to_head_unchecked::h62ad5a776ae54d00@http://localhost:8080/pkg/package_bg.wasm:wasm-function[131]:0x27bc3
seed_style_preview::style::add_css_to_head::h60ad31a0169818cf@http://localhost:8080/pkg/package_bg.wasm:wasm-function[517]:0xbfbc5
<seed_style_preview::style::Style as seed_style_preview::style::LocalUpdateEl<seed::virtual_dom::node::el::El<Ms>>>::update_el::h539ccf2f2eea2de6@http://localhost:8080/pkg/package_bg.wasm:wasm-function[136]:0x3b8ff
app_name::ui::view::h5167344618b77e89@http://localhost:8080/pkg/package_bg.wasm:wasm-function[481]:0xb9a9c
seed::app::App<Ms,Mdl,INodes,GMs>::rerender_vdom::hdff5c3ed2da3b841@http://localhost:8080/pkg/package_bg.wasm:wasm-function[160]:0x56e0a
seed::app::App<Ms,Mdl,INodes,GMs>::run::h2df28b1cf910dd71@http://localhost:8080/pkg/package_bg.wasm:wasm-function[155]:0x53195
seed::app::App<Ms,Mdl,INodes,GMs>::start::h99cac74016595b38@http://localhost:8080/pkg/package_bg.wasm:wasm-function[320]:0x998db
app_name::start::h373e38ca4fd48382@http://localhost:8080/pkg/package_bg.wasm:wasm-function[3483]:0x172027
start@http://localhost:8080/pkg/package_bg.wasm:wasm-function[7484]:0x1bd616
init@http://localhost:8080/pkg/package.js:900:10
) "\n.sst-class.seedstyle--3y4Ppn7om4X6p:after(content to put after){\n}\n" package.js:696:17
```

This issue was discussed with @rebo via Discord.

**Comment by eopb** [comment]
Closing as fixed in 0f23afafe29517819d8e99f5381fefec0d6cc41b

**Comment by rebo** [comment]
Just a note if anyone is getting this error still.

Although `after()` and `before()` now produce the correct css pseudo selector. I.e. `::after{ ...}` you will still receive an error log in the console if you do not escape enclosing quotes on the a string argument to `contents()`.

This is because the css outputed is `contents: #string_value_here`. this is to allow for instance `contents: url('www.blah')` to be representable with a `&str` argument to `contents()`. This is consistent with all the other CSS values.

At some point might reconsider the above point.
